src/operator/nn/concat.cc

```diff
@@ -261,9 +278,7 @@ Example::
 .set_attr<nnvm::FInferType>("FInferType", ConcatType)
 .set_attr<FInferStorageType>("FInferStorageType", ConcatForwardInferStorageType)
 .set_attr<FCompute>("FCompute<cpu>", ConcatCompute<cpu>)
-#if MXNET_USE_MKLDNN == 1
```

Comment:
pls update doc

Reply:
Have updated several docs and add sparse op alias.